.github/workflows/validate.yml

```diff
@@ -43,8 +43,12 @@ jobs:
           # lot of segfaults caused by ghc bugs
           - os: "windows-latest"
             ghc: "8.8.4"
+	  # it also throws segfaults randomly
           - os: "windows-latest"
             ghc: "8.4.4"
+	  # it often randomly does "C:\Users\RUNNER~1\AppData\Local\Temp\ghcFEDE.c: DeleteFile "\\\\?\\C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\ghcFEDE.c": permission denied (Access is denied.)"
```

Comment:
seems to be some mixed tabs/spaces here? (but maybe github diff rendering is broken)

Reply:
You are right. Will fix next time. :)

Reply:
Huh, the tabs might have broken the script, making CI ignore all the validate jobs. I've removed the tabs now. Who knew?